decidim-system/spec/forms/decidim/system/update_organization_form_spec.rb

```diff
@@ -103,7 +103,7 @@ module Decidim::System
             omniauth_settings_facebook_app_id: Decidim::AttributeEncryptor.encrypt("foo")
           },
           file_upload_settings: {
-            allowed_file_extensions: {
+            :allowed_file_extensions => {
```

Comment:
why not changing the other keys to symbols ?

Reply:
Done in 984ace2000ad97e768bb8eba091d4ba7eda2ac11